swagger.yaml

```diff
@@ -222,6 +228,7 @@ paths:
       summary: "Get a list of datasets"
       description: "Returns a list of all datasets provided by the ONS that can be filtered using the filter API"
       parameters: 
+      - $ref: '#/parameters/is-based-on'
       - $ref: '#/parameters/limit'
       - $ref: '#/parameters/offset'
       produces:
```

Comment:
fyi - not sure if this has already been implemented but cannot see is_based_on field against the response model for a dataset resource. I know that wasn't your focus but might be worth adding in?

Reply:
I've added it to the Dataset resource so it's now being returned